app/src/main/java/tv/vizbee/demo/network/ApiInterface.kt

```diff
@@ -0,0 +1,27 @@
+package tv.vizbee.demo.network
+
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.Header
+import retrofit2.http.POST
+import retrofit2.http.Path
+import tv.vizbee.demo.model.LoginRequest
+import tv.vizbee.demo.model.LoginResponse
+import tv.vizbee.demo.model.RegCodeStatusRequest
+
+interface ApiInterface {
```

Comment:
Please think about better names. It’s generic.

Reply:
Changed